### ISSUE TITLE
update "Trigger registry scans with webhooks" page

### ISSUE THREAD

**Issue by alexshkatov** [twistlock/docs#213]
Seem like the `Trigger registry scans with webhooks` page (https://github.com/twistlock/docs/blob/master/admin_guide/vulnerability_management/registry_scanning/webhooks.adoc) is outdated.

`Allow image scans initiated by registry webhooks` option was deprecated in 2018.

**Comment by sgordon46** [comment]
@alexshkatov deprecated in 2018 for a specific registry?

**Comment by alexshkatov** [comment]
@sgordon46 No, for any registry. Starting from the commit https://github.com/twistlock/twistlock/commit/091d4e6e3606437573f468294567c3874156eabe (Jul 11, 2018), 
registry web hooks are always enabled (no need to turn it ON)

![Screenshot from 2020-11-02 16-23-10](https://user-images.githubusercontent.com/26891178/97878785-c02c7080-1d27-11eb-96a1-3f7575075fb1.png)

**Comment by sgordon46** [comment]
@alexshkatov ok, this makes more sense. thanks for the clarification.
